docs/customization/pages.rst

```diff
@@ -141,3 +142,101 @@ To extract a new field for one or more websites:
 
             def parse_product(self, response: DummyResponse, product: CustomProduct):
                 yield from super().parse_product(response, product)
+
+.. _fix-search:
+
+Fixing search support
+=====================
+
+If the default implementation to build a request out of :ref:`search queries
+<search-queries>` does not work on a given website, you can implement your
+own search request page object to fix that. See
+:ref:`custom-request-template-page`.
+
+For example:
+
+.. code-block:: python
+
+    from web_poet import handle_urls
+    from zyte_common_items import SearchRequestTemplatePage
+
+
+    @handle_urls("example.com")
+    class ExampleComSearchRequestTemplatePage(SearchRequestTemplatePage):
+        @field
+        def url(self):
+            return "https://example.com/search?q={{ keyword|quote_plus }}"
+
+
+Reusing the default implementation
+----------------------------------
```

Comment:
I still prefer not to document it, at least here. It would allow us to change the implementation without breaking the documented ways to use the spider. What do you think about moving it to the module docstring?